spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/PoolControllerHelperService.java

```diff
@@ -398,6 +398,27 @@ public JSONObject printPoolBarcodes(HttpSession session, JSONObject json) {
       return JSONUtils.SimpleJSONError("Cannot print barcodes: " + e.getMessage());
     }
   }
+  
+  public JSONObject changePoolIdBarcode(HttpSession session, JSONObject json) {
+    Long poolId = json.getLong("poolId");
+    String idBarcode = json.getString("identificationBarcode");
+    
+    try {
+      if (!"".equals(idBarcode)) {
+        Pool pool = requestManager.getPoolById(poolId);
```

Comment:
This causes the warning `Plate is a raw type. References to generic type Plate<T,S> should be parameterized`.